modules/monitoring-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -6,7 +6,7 @@
 [id="creating-alerting-rules-for-user-defined-projects_{context}"]
 = Creating alerting rules for user-defined projects
 
-You can create alerting rules for user-defined projects. Those alerting rules will fire alerts based on the values of chosen metrics.
+You can create alerting rules for user-defined projects. Those alerting rules will fire alerts based on the values of the chosen metrics.
```

Comment:
```suggestion
You can create alerting rules for user-defined projects. Those alerting rules <report> or <send> alerts based on the values of the chosen metrics.
```

See source above for suggestions on "Fire". Also, note use of future tense here: https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense

Reply:
Changed to `trigger`